cmd/kad/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/node"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "inspector",
+		Short: "CLI for interacting with Storj Kademlia network",
+	}
+	getNodeCmd = &cobra.Command{
+		Use:   "get",
+		Short: "get node with `id`",
+		RunE:  GetNode,
+	}
+)
+
+// Inspector gives access to kademlia and overlay cache
+type Inspector struct {
+	overlay  overlay.Client
+	identity *provider.FullIdentity
+}
+
+func identity() (*provider.FullIdentity, error) {
+	ca, err := provider.NewTestCA(context.Background())
```

Comment:
why do we use a testing ca?

Reply:
I swapped this for `node.NewFullIdentity`